pkg/component/worker/nllb/envoy.go

```diff
@@ -242,11 +241,11 @@ func writeEnvoyConfigFiles(params *envoyParams, filesParams *envoyFilesParams) e
 			return bufferedWriter.Flush()
 		})
 		if err != nil {
-			errs = multierr.Append(errs, fmt.Errorf("failed to write %s: %w", fileName, err))
+			errs = append(errs, fmt.Errorf("failed to write %s: %w", fileName, err))
```

Comment:
Perhaps for another PR since that would change behavior

Reply:
> does it make sense to write the second file if writing the first already fails instead of just returning the error when it happens?

That's a good question. I think yes, it makes sense, since the templated values in those files don't depend on each other. The first defines the local bind addresses, the second defines the remote endpoints. So in any case the resulting config should remain valid, albeit partially out of date.